package.json

```diff
@@ -6,7 +6,9 @@
     "build": "sass src/main/resources/scss/styles.scss src/main/resources/io/jenkins/plugins/designlibrary/styles.css && postcss src/main/resources/io/jenkins/plugins/designlibrary/styles.css --replace --use autoprefixer",
     "dev": "sass --watch src/main/resources/scss/styles.scss src/main/resources/io/jenkins/plugins/designlibrary/styles.css && postcss src/main/resources/io/jenkins/plugins/designlibrary/styles.css --replace --use autoprefixer",
     "mvntest": "npm run test",
-    "test": "exit 0"
+    "test": "exit 0",
+    "lint": "prettier --check .",
+    "lint:fix": "prettier --write ."
```

Comment:
Usually linting is reserved for something else. It is preferable to use "format" instead of "lint" here.

Reply:
linting includes formatting, you just want one fix command, this is fine.